drivers/power/ltc4162l/iio_ltc4162l.c

```diff
@@ -0,0 +1,514 @@
+/***************************************************************************//**
+ *   @file   iio_ltc4162l.c
+ *   @brief  Implementation of IIO LTC4162L Driver.
+ *   @author Marc Paolo Sosa (marcpaolo.sosa@analog.com)
+********************************************************************************
+ * Copyright 2024(c) Analog Devices, Inc.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of Analog Devices, Inc. nor the names of its
+ *    contributors may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES, INC. “AS IS” AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO
+ * EVENT SHALL ANALOG DEVICES, INC. BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
+ * OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+ * EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+
+#include <errno.h>
+#include "iio_ltc4162l.h"
+#include "ltc4162l.h"
+#include "no_os_alloc.h"
+#include "no_os_error.h"
+
+#define LTC4162L_IIO_CH_ATTR_RW(_name, _priv)	\
+	{					\
+		.name = _name,			\
+		.priv = _priv,			\
+		.show = ltc4162l_ch_attr_show,	\
+		.store = ltc4162l_ch_attr_store,	\
```

Comment:
The newline characters are not aligned.

What i do is put a whitespace:
```
#define LTC4162L_IIO_CH_ATTR_RW(_name, _priv) \
	{ \
		.name = _name, \
		.priv = _priv, \
		.show = ltc4162l_ch_attr_show, \
		.store = ltc4162l_ch_attr_store, \
	}
```

There is nothing that can ever go wrong with this approach, no alignment to care about.